### ISSUE TITLE
Test for WCAG technique H83: Using the target attribute to open a new window on user request and indicating this in link text

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#118]
http://www.w3.org/TR/WCAG20-TECHS/H83
The objective of this technique is to avoid confusion that may be caused by the appearance of new windows that were not requested by the user. Suddenly opening new windows can disorientate or be missed completely by some users. In HTML 4.01 Transitional and XHTML 1.0 Transitional, the target attribute can be used to open a new window, instead of automatic pop-ups. (The target attribute is deleted from HTML 4.01 Strict and XHTML 1.0 Strict.) Note that not using the target allows the user to decide whether a new window should be opened or not. Use of the target attribute provides an unambiguously machine-readable indication that a new window will open. User agents can inform the user, and can also be configured not to open the new window. For those not using assistive technology, the indication would also be available from the link text.

Procedure
1. Activate each link in the document to check if it opens a new window.
2. For each link that opens a new window, check that it uses the target attribute.
3. Check that the link text contains information indicating that the link will open in a new window.

Expected Results
- Checks #2 and #3 are true.
